### ISSUE TITLE
job recovery

### ISSUE THREAD

**Issue by Imoteph** [PacificBiosciences/FALCON#58]
Hi,

I was wondering how job recovery works. 
Currently Falcon runs the DAligner Step and I like to stop/kill the process and change the SGE parameters.
Can I just run fc_run.py and it will restart the last jobs where it crashed?

**Comment by pb-jchin** [comment]
@Imoteph I put some tips at https://github.com/PacificBiosciences/FALCON/wiki#tips-for-restarting-jobs-with-falcon 

Those are simple cases.  There are some case which is more complicated and only useful when you don't want to repeat some calculation.  For example, if there are some corrupted daligner output files, I might just want to redo the alignment for the corrupted block-block comparison. (Yes, when I developed the code, our storage infrastructure could not handle the load, so I got quite a few corrupted files.)  In such case, you will need to know the internal work for both Falcon and Daligner to figure out which block-block comparison is corrupted, manually patching them, setting right sentinel files, then run `fc_run.py` to resume the jobs.
